tests/neg/i11797.scala

```diff
@@ -0,0 +1,6 @@
+object Foo:
+  @annotation.implicitNotFound("Oops")
+  type Bar
```

Comment:
Do we want to also add a test with an `opaque type`? Are `opaque types` just represented as a `TypeRef` like transparent types?

Reply:
yes that's the same